pkgs/applications/misc/resp-app/default.nix

```
@@ -0,0 +1,116 @@
+{ stdenv
+, mkDerivation
+, lib
+, fetchFromGitHub
+, brotli
+, lz4
+, pyotherside
+, python3
+, python3Packages
+, qtbase
+, qtcharts
+, qtdeclarative
+, qtgraphicaleffects
+, qtimageformats
+, qmake
+, qtquick1
+, qtquickcontrols2
+, qtsvg
+, qttools
+, snappy
+, wrapQtAppsHook
+, zstd
+}:
+
+let
+  redis-rdb-tools = with python3Packages; buildPythonPackage rec {
+    pname = "redis-rdb-tools";
+    version = "resp_rdbtools-0.1.15";
```

```suggestion
    version = "0.1.15";
```
Can we move this to pythonPackages?

---

Found out it's already packaged as `rdbtools` and only needs a patch to work with `RESP.app`